allocations/models.py

```diff
@@ -0,0 +1,28 @@
+from django.db import models
+import json
+import logging
+from django.conf import settings
+
+logger = logging.getLogger('allocations')
+
+class Allocation(models.Model):
+    STATUS = (
+        ('Inactive', 'Expired or overused'),
```

Comment:
what about making these uncapitalized? It makes it easier to normalize them when storing:

```python
status = req.params['status'].lower()
```